packages/block-library/src/social-link/index.php

```diff
@@ -15,7 +15,7 @@
  * @return string Rendered HTML of the referenced block.
  */
 function render_block_core_social_link( $attributes, $content, $block ) {
-	$open_in_new_tab = $block->context['openInNewTab'];
+	$open_in_new_tab = ( isset( $block->context['openInNewTab'] ) ) ? $block->context['openInNewTab'] : false;
```

Comment:
Would it be fine to remove parenthesis around `isset` call?
```suggestion
	$open_in_new_tab = isset( $block->context['openInNewTab'] ) ? $block->context['openInNewTab'] : false;
```